css/wishlist.css

```diff
@@ -0,0 +1,95 @@
+.wishlist-wrapper {
+  margin-top: 4rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.wishlist-cards {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+}
+
+.card {
+  background-color: #eee;
+  width: 14rem;
+  display: flex;
+  flex-direction: column;
+  margin: 1rem;
+}
+
+.card-image {
+  position: relative;
+  width: 100%;
+  height: 280px;
+}
+.card-image img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+.card-badge {
+  margin-top: 1rem;
+  background-color: #9646f1;
+  color: white; /*not decided yet, will use variable later*/
+  position: absolute;
+  left: 0rem;
+  top: 0.5rem;
+  padding: 0.1rem 0.4rem;
+}
+.card-remove {
+  font-weight: 500;
+  color: #ffffff; /*not decided yet, will use variable later*/
+  position: absolute;
+  right: 0.08rem;
+  top: 1rem;
+}
+/*card content*/
+.card-info {
+  display: flex;
+  flex-direction: column;
+}
+
+.card-title {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0.5rem;
+}
+
+.card-title i {
+  margin-top: -1rem;
+  color: var(--color-primary);
+  cursor: pointer;
+}
+
+.card-price {
+  display: flex;
+  justify-content: flex-start;
+  gap: 0.4rem;
+  padding-left: 0.6rem;
+}
+
+.card-price .og-price {
+  text-decoration: line-through;
+  color: grey;
+}
+.card-price .disc-percent {
+  color: var(--color-success);
+}
+.card button {
+  cursor: pointer;
+  border: none;
+  background-color: black;
+  color: var(--color-white);
+  margin: 0.2rem 0.4rem;
+  margin-bottom: 0.6rem;
```

Comment:
if possible you can use margin: 0.2rem 0.4rem 0.6rem 0.4rem instead of line 92 and 93(i think this will work if not then correct me)

Reply:
I have a hard time remembering the top right bottom left syntax, but I think my code is redundant. Will make these changes and add a comment, thanks for pointing it out